interfaces/builtin/upower_observe.go

```diff
@@ -20,7 +20,12 @@
 package builtin
 
 import (
+	"bytes"
+	"fmt"
+
 	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/release"
+	"github.com/snapcore/snapd/snap"
 )
 
 const upowerObserveConnectedPlugAppArmor = `
```

Comment:
@jdstrand Yes that is what I am doing here but I don't understand why we have to implement the same slot policy in both interfaces. A snap implementing the upower-observe slot should always implement the upower-observe interface as well. So both will always go together also because of the snaping (upower- prefix). It only differentiates on the plug side which one will be used.